broker-daemon/state-machines/order-state-machine.spec.js

```diff
@@ -268,18 +268,11 @@ describe('OrderStateMachine', () => {
       await osm.create(blockOrderId, params)
 
       expect(Order).to.have.been.calledWith(blockOrderId, sinon.match(params))
-    })
-
-    it('gets the makerAddress for the order', async () => {
-      await osm.create(blockOrderId, params)
-
       expect(getPaymentChannelNetworkAddressStub).to.have.been.calledTwice()
-      expect(osm.order.makerBaseAddress).to.be.equal('bolt:adsfsdf')
```

Comment:
why are we getting rid of this test?

Reply:
Because `Order` is being stubbed so it never gets set due to a change in the code; before it was being set in `create` and now it's being set in the `Order` constructor, which isn't being run due to the stub.